.azuredevops/platformPipelines/platform.deployment.removal.yml

```
@@ -0,0 +1,125 @@
+name: '.Platform - Remove Deployments'
+
+parameters:
+  - name: handleSubscriptionScope
+    displayName: Include Subscription deployments
+    type: boolean
+    default: true
+  - name: handleManagementGroupScope
+    displayName: Include Management Group deployments
+    type: boolean
+    default: true
```

should this be set to true by default? what if service connection does not have permissions over MG?

---

We must. Otherwise our own pipeline won't solve our own problem. Users can set it to false in their own environment if they don't have the permissions

---

I'd then suggest a note such as 
```suggestion
    default: true # Note: This requires your service principle to have permissions over management group scope.
```

and clarifying that better in the docs.

---

Agreed